divisible.rb

```diff
@@ -0,0 +1,14 @@
+#using each method
+(1..100).each do 
+    #insert block with argrument to see if number is divisible by 2,3, or 5
+      |number|
+      if number % 2==0
+       puts"#{number} is divisible by 2"
```

Comment:
The assignment asks that you "Write a method that returns in an array" all the numbers that you have printed here. While I see that you're _using_ the built-in `Range#each` method and printing out all the numbers, the assignment is expecting you to build an array _as you're iterating_ through the numbers, and then return it at the end of the method that _you_ define (using the `def` keyword). Could you do that here?